### DISCUSSION TITLE
Set Tags - Sort list alphabetically

### DISCUSSION THREAD

**Discussion by StereotypicalCat** [jesec/flood#135] [Ideas]
When setting tags of a torrent in flood by doing:
* Select torrent
* Right Click
* Set Tags
* Open List of Tags
The list of tags is not sorted alphabetically. Having this sorted alphabetically would be awesome!

**Comment by jesec** [comment]
Good idea.

**Comment by jesec** [comment]
Implemented by https://github.com/jesec/flood/commit/7d44626451db2d4825e1265392a45e23a7aa47d7.
